lib/Cake/View/View.php

```diff
@@ -455,7 +455,7 @@ public function elementExists($name) {
  * a plugin view/layout can be used instead of the app ones. If the chosen plugin is not found
  * the view will be located along the regular view path cascade.
  *
- * @param string $view Name of view file to use
+ * @param bool|string $view Name of view file to use.
```

Comment:
use of `bool` for first args  is useless 
`bool` is not acceptable here

Reply:
Well, that's how it is used: `$this->render(false, $layout);`
https://github.com/cakephp/cakephp/blob/2.x/lib/Cake/Controller/Controller.php#L1017

And passed unaltered from Controller to View here - https://github.com/cakephp/cakephp/blob/2.x/lib/Cake/Controller/Controller.php#L963.

Reply:
OK, so just `string|false` is good
and no needed to `!is_bool($view) &&`  (line 470)